content/ytt/js/app.css

```diff
@@ -105,6 +105,7 @@
 
 .example-set.expanded {
   display: block;
+  background: #fffff4;
```

Comment:
This color is _in theme_. 

However, by my eye, a very pale yellow looks washed out against the white body.

I sketched this in the browser:

![image](https://user-images.githubusercontent.com/12101915/117671038-43a69e80-b15d-11eb-9882-101235f9b1f8.png)

It involved:
```css
#playground p {
  margin-block-end: 0px;
}

.example-set.expanded {
    border: 1px;
    border-color: lightgray;
    border-style: solid;
    padding-left: 12px;
    padding-right: 12px;
    padding-top: 8px;
}
```

and removed the `h3.example-set-name`

Thoughts?